kanidmd/src/lib/idm/server.rs

```diff
@@ -188,6 +191,17 @@ impl IdmServer {
             rp_id,
         });
 
+        // Setup our auth token signing key.
+        let bundy_handle = HS512::generate_key()
+            .and_then(|bundy_key| HS512::from_str(&bundy_key))
+            .map_err(|e| {
+                ladmin_error!(au, "Failed to generate uat_bundy_hmac - {:?}", e);
+                OperationError::InvalidState
+            })?;
+        let uat_bundy_hmac = Arc::new(CowCell::new(bundy_handle));
```

Comment:
Why did this get moved down?

Reply:
because it turned out there was a bunch of webauthn setup logic above, and this was added smack in the middle of it and didn't make sense, so itmade logical sense to move it later.